src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,45 @@
+package command;
```

Comment:
Just curious, why did you package into command and not something like duke.command or duke.ui? The coding standard seems to recommend the below: 

> For school projects, the root name of the package should be your group name or project name followed by logical group names.e.g. todobuddy.ui, todobuddy.file etc.

Reply:
Must have missed that out. I'll do it for the future increments, thank you.